sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/rules/QueryExecutionMetering.scala

```diff
@@ -95,3 +107,26 @@ case class QueryExecutionMetering() {
      """.stripMargin
   }
 }
+
+case class QueryExecutionMetrics(
+    time: Long,
+    numRuns: Long,
+    numEffectiveRuns: Long,
+    timeEffective: Long) {
+
+  def +(metrics: QueryExecutionMetrics): QueryExecutionMetrics = {
```

Comment:
is this method needed?

Reply:
Emm. Not used by now, agree to add it once we really need this. Removed in 36bf2e134436a9b5c7d96c839855dd69cef45538.